test/unit/has-yarn-test.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+const os = require('os');
+const fixturify = require('fixturify');
+const { describe, it } = require('../helpers/mocha');
+const { expect } = require('../helpers/chai');
+const hashYarn = require('../../src/has-yarn');
+
+describe(hashYarn, function() {
+  beforeEach(function() {
+    this.tempDir = os.tmpdir();
```

Comment:
Do you mind following the convention of `tmpPath = await createTmpDir();` for this?